client/my-sites/checkout/src/components/checkout-sidebar-plan-upsell/index.tsx

```diff
@@ -17,7 +20,6 @@ import {
 	getItemVariantDiscountPercentage,
 } from '../item-variation-picker/util';
 import './style.scss';
-
```

Comment:
Since I mentioned it for that one spacing removal and you fixed it, I figured I'd mark the other places this space was removed.